alluxio-ui/master/src/containers/App/__snapshots__/App.test.tsx.snap

```diff
@@ -1153,6 +1153,13 @@ exports[`App Connected component Matches snapshot 1`] = `
                           "innerText": "Resources",
                           "url": "https://alluxio.org/alluxio-resources/",
                         },
+                        Object {
+                          "attributes": Object {
+                            "target": "_blank",
+                          },
+                          "innerText": "Slack Channel",
+                          "url": "https://slackin.alluxio.org/",
```

Comment:
can we use "https://www.alluxio.org/slack"? slackin is the implementation details to distribute slack invitation code

Reply:
https://www.alluxio.org/slack redirects to https://slackin.alluxio.org. I don't think it makes a difference

Reply:
I'm ok with either way. I'll make @apc999's suggestion happen.